README.md

```diff
@@ -414,6 +414,18 @@ The current value of the setting is global in scope (i.e. applies to all
 sessions, whenever created) and is always consulted for the creation of
 any data object handle to govern that handle's cleanup behavior.
 
+Also, alternatively, the client may opt in to a special "redirect" behavior
```

Comment:
`in to` -> `into` ?

Hmm ...

Reply:
I think so.  Was probably thinking in terms of "opt-in to" something.

correcting....